Support/Testing/CircleCI/Dockerfile

```diff
@@ -65,7 +95,9 @@ RUN update-alternatives --set x86_64-w64-mingw32-gcc /usr/bin/x86_64-w64-mingw32
 RUN update-alternatives --set x86_64-w64-mingw32-g++ /usr/bin/x86_64-w64-mingw32-g++-posix
 
 # Install test dependencies
-RUN apt-get install -y git lldb-6.0 gdb clang-format-6.0 make dejagnu
+RUN apt-get install -y git lldb-7 gdb clang-format-7 make dejagnu
+RUN ln -s lldb-7 /usr/bin/lldb
```

Comment:
Ditto.

Reply:
Yea but I want to remove `clang-6.0` and `lldb-6.0` from usages throughout the project instead of manually having to update them every llvm release.